benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py

```diff
@@ -0,0 +1,305 @@
+
```

Comment:
Might want to split this into separate classes for each resource type.

Reply:
Right, I will split it

Reply:
@RobertKrawitz Thx again for this comment, fixed